internal/server/openapi.yaml

```diff
@@ -0,0 +1,1253 @@
+        text/plain:
+          type: string
+          example: Conflict
+    badRequestError:
+      description: Your request was invalid
+      content:
+        text/plain:
+          type: string
+          example: Bad Request
+  securitySchemes:
+    BearerAuth:
+      type: http
+      scheme: bearer
+      bearerFormat: JWT
+  schemas:
+    teamKey:
+      type: string
+      example: frc2733
+    eventKey:
+      type: string
+      example: 2018pncmp
+    matchKey:
+      type: string
+      example: qm30
+    realm:
+      required:
+        - name
+        - shareReports
+        - id
+      properties:
+        name:
+          type: string
+          example: Pigmice
+        shareReports:
+          type: boolean
+          example: true
+        id:
+          type: integer
+          format: int64
+          example: 1
+    reportStat:
+      required:
+        - name
+        - attempts
+        - successes
+      properties:
+        name:
+          type: string
+          example: Sandstorm 2
+        attempts:
+          type: integer
+          example: 9
+        successes:
+          type: integer
+          example: 2
+    report:
+      required:
+        - autoName
+        - data
+      properties:
+        reporterId:
+          $ref: '#/components/schemas/id'
+        autoName:
+          type: string
+          example: Center Scale
+        data:
+          $ref: '#/components/schemas/reportData'
+    reportData:
+      required:
+        - auto
+        - teleop
+      properties:
+        auto:
+          type: array
+          items:
+            $ref: '#/components/schemas/reportStat'
+        teleop:
+          type: array
+          items:
+            $ref: '#/components/schemas/reportStat'
+    match:
+      required:
+        - key
+        - time
+        - redAlliance
+        - blueAlliance
+      properties:
+        key:
+          $ref: '#/components/schemas/matchKey'
+        time:
+          type: string
+          format: date-time
+          example: '2018-04-06T23:21:38Z'
+        scheduledTime:
+          type: string
+          format: date-time
+          example: '2018-04-06T23:18:00Z'
+        redScore:
+          type: integer
+          example: 426
+        blueScore:
+          type: integer
+          example: 265
+        redAlliance:
+          type: array
+          items:
+            type: string
+            example: frc5468
+        blueAlliance:
+          type: array
+          items:
+            type: string
+            example: frc4488
+    stats:
+      type: array
+      items:
+        required:
+          - attempts
+          - successes
+          - name
+        properties:
+          attempts:
+            $ref: '#/components/schemas/statPoint'
+          successes:
+            $ref: '#/components/schemas/statPoint'
+          name:
+            type: string
+            example: Cubes on Scale
+    statPoint:
+      required:
+        - max
+        - avg
```

Comment:
can you add a `type` property here? If max is between 0 and 1 we don't know to display it as a percentage or as a decimal

`type: 'number'|'percent'`

Reply:
done